seda/assetlist.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "seda",
+  "assets": [
+    {
+      "description": "The SEDA Token (SEDA), previously known as Flux Token (FLX), serves three distinct purposes: governance, participation, and network utilization.",
```

Comment:
Can we add an `extended_description` that reads "SEDA is a standard for modular data transport and querying. Any data type, for all networks."